play/src/front/Components/Input/RangeSlider.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+    export let id: string | undefined = undefined;
+    export let label: string;
+    export let value = 0;
+    export let min = 0;
+    export let max = 100;
+    export let onChange = (v: number) => {};
+    //secondary = bleu   light = noir sinon par défaut la ligne est blanche
+    export let variant: "secondary" | "light" | "" = "";
+    export let buttonShape: "square" | "" = "";
+
+    let uniqueId = id || `input-${Math.random().toString(36).substr(2, 9)} `;
+</script>
+
+<label for={uniqueId} class=""> {label} <slot />: {value}%</label>
```

Comment:
We could add a "unit" prop. The value is not necessarily expressed in %.

```
export let unit = "%";
```

```suggestion
<label for={uniqueId} class=""> {label} <slot />: {value}{unit}</label>
```